src/auth/project-config.ts

```diff
@@ -86,6 +102,11 @@ export class ProjectConfig {
     if (typeof request.smsRegionConfig !== 'undefined') {
       SmsRegionsAuthConfig.validate(request.smsRegionConfig);
     }
+
+    // Validate Multi Factor Config if provided
+    if (typeof request.mfa !== 'undefined') {
```

Comment:
Yes, thanks for catching this miss. Looks like it did not break any tests because the validation function is being tested out separately and this if statement would always be false so the code block wouldn't have been executed. The validKeys has `multiFactorConfig` as the required key in request so the `mfa` field wasn't caught there either.

Reply:
Hmm.. shouldn't it have failed the tests where invalid provider config was being passed in, like the unit tests with `const invalidProviderConfigs = [NaN, 0, 1, '', 'a', {}, { a: 1 }, _.noop, true, false,]`?

Reply:
@pragatimodi

Reply:
Well, invalidProviderConfigs was not tested because the field itself was marked as `nil` considering the `mfa` values weren't set.